### PR TITLE
Acceleration branch

### PR DESCRIPTION
**Pull request by FrederikSchibelfeldt**

### CODE REVIEW

File: src/classes.py [hunk 1]
```diff
@@ -32,46 +36,77 @@ def switch_lanes(self,left_back,right_back,left_front,right_front): #params here
...
+                a = ai * (1 - (v / v0) ** 4) - ai * (
+                            ((s0 + v * t) / si) + ((v * deltav) / (2 * si * math.sqrt(ai * b)))) ** 2
+
+                adjust = (self.current_speed - self.front / 1) + 10  # adjust
+                if (self.current_speed - adjust) < 0: adjust = self.current_speed
                 action = "brake"
-                params = {"value":adjust}
+                params = {"value": a}
 
-        elif(self.front > self.speed_limit + 20 and self.random_mood() > self.mood):
```
Comment (mactat) [inline src/classes.py:50]: Different condition needed. Tested with `(self.front > 20 and self.random_mood() > self.mood - 0.1)` works better
multiple

**Review by mactat — COMMENTED** [review]
Minor changes, but overall looks fine. Thanks for refactoring the code 👍

**Comment by mactat** [comment]
[Fix acceleration](https://trello.com/c/qVBXBzcW/70-fix-acceleration)

**Comment by mactat** [comment]
[Fix deceleration of the car](https://trello.com/c/9U8Jnn8l/57-fix-deceleration-of-the-car)

File: src/classes.py [hunk 2]
```diff
@@ -32,46 +36,77 @@ def switch_lanes(self,left_back,right_back,left_front,right_front): #params here
...
+        # * t = reaction time
+        # * s0 = Desired minimum distance between cars
+        # * si = Distance to car in front
+        # * v = Current speed of car
+        # * ai = Max acceleration of car, taken from average acceleration in m/s^2 of cars
+        # * b = Comfortable deaccleration of car, taken from google search
+        # * v0 = desired speed of car, set as the speed limit
+        deltav = self.current_speed - self.frontSpeed
+        t = self.reaction_time
+        s0 = 3
```
[the remark this answers is not in the document]
Comment (FrederikSchibelfeldt) [reply src/classes.py]: Can be played around with, i found around 5 to be best

**Review by mactat — APPROVED** [review]

**Pull request merged**